### ISSUE TITLE
VueCLI wont install.

### ISSUE THREAD

**Issue by Wishingirl** [vuejs/vue-cli#5922]
### Version
4.5.6

### Environment info
```

Environment Info:

  System:
    OS: Windows 10 10.0.18362
    CPU: (4) x64 Intel(R) Core(TM) i5-2400 CPU @ 3.10GHz
  Binaries:
    Node: 14.11.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.5 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 6.14.8 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Chrome: Not Found
    Edge: Spartan (44.18362.449.0)
  npmGlobalPackages:
    @vue/cli: Not Found


```

### Steps to reproduce
yarn global add @vue/cli-init

vue init nuxt-community/express-template test

### What is expected?
To make the template for nuxt vuejs

### What is actually happening?
it says vuecli it's not installed even tough i had just installed it.

![image](https://user-images.githubusercontent.com/71225598/94950217-8a0a8f80-04e2-11eb-82d2-cb0fda34058e.png)

**Comment by gregg-cbs** [comment]
According to the docs its not cli-init, just cli
`yarn global add @vue/cli`

**Comment by fangbinwei** [comment]
It may be the problem of your yarn

**Comment by fangbinwei** [comment]
I guess you install `@vue/cli` by npm, but the warning message reminds you to install by `yarn global add`. It has been fixed in `4.5.8`, https://github.com/vuejs/vue-cli/pull/5871. After upgrade `@vue/cli`, the warning message will remind you to use `npm install -g`

Although Vue CLI will try to require module from both  yarn global path and npm global path by dependencies `import-global`,  `global-dirs@0.1.1`. It seems the `global-dirs@0.1.1` give the wrong yarn global path(fixed in 1.0.0 https://github.com/sindresorhus/global-dirs/issues/3 )

Test in my windows, 
my yarn global path is `%LOCALAPPDATA%\Yarn\Data\global`, but the value return by `global-dirs` is `%LOCALAPPDATA%\Yarn\config\global`.

**Comment by Wishingirl** [comment]
Thanks its running now.

**Comment by fangbinwei** [comment]
> I guess you install @vue/cli by npm

That's true ?
@Wishingirl

**Comment by Wishingirl** [comment]
I just checked seems i still have trouble..
![image](https://user-images.githubusercontent.com/71225598/97193529-6668fb00-17a9-11eb-8705-aba164a3a2a0.png)

**Comment by Wishingirl** [comment]
Env info:
```
04:35:53 user@WINDOWS-PC ExpressVue →yarn global dir
C:\Users\user\AppData\Local\Yarn\Data\global

```

**Comment by fangbinwei** [comment]
What the output information of  `yarn global list` and `npm list -g --depth=0` @Wishingirl

**Comment by Wishingirl** [comment]
![image](https://user-images.githubusercontent.com/71225598/97194510-851bc180-17aa-11eb-8ffa-35fbeecb850d.png)

**Comment by fangbinwei** [comment]
You install `@vue/cli` by both yarn and npm. If the output of  `vue --version` is `4.5.6`, It means you executed `vue` from npm global bin path.

So, try to run `npm uninstall @vue/cli -g` and use the version installed by yarn. @Wishingirl

**Comment by Wishingirl** [comment]
Alright. On it.

**Comment by Wishingirl** [comment]
It works  now! Generating templates and working as intended. Thanks.

**Comment by fangbinwei** [comment]
> I guess you install `@vue/cli` by npm, but the warning message reminds you to install `@vue/cli-init` by `yarn global add`. It has been fixed in `4.5.7`, #5871. After upgrade `@vue/cli`, the warning message will remind you to use `npm install -g`
> 
> Although Vue CLI will try to require module from both yarn global path and npm global path by dependencies `import-global`, `global-dirs@0.1.1`. It seems the `global-dirs@0.1.1` give the wrong yarn global path(fixed in 1.0.0 [sindresorhus/global-dirs#3](https://github.com/sindresorhus/global-dirs/issues/3) )
> 
> Test in my windows,
> my yarn global path is `%LOCALAPPDATA%\Yarn\Data\global`, but the value return by `global-dirs@0.1.1` is `%LOCALAPPDATA%\Yarn\config\global`.


Since `@vue/cli@^4.5.7` can output more accurate warnings message, `import-global` should be removed or try to upgrade `global-dirs`. Still need to be considered.
